transformers/integration_tests/deny_test.go

```diff
@@ -0,0 +1,115 @@
+package integration_tests
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/deny"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/libraries/shared/fetcher"
+	"github.com/makerdao/vulcanizedb/libraries/shared/transformer"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Deny transformer", func() {
+	const (
+		defaultOffset int = 0
+		vatOffset     int = -1
+	)
+
+	Context("Cat deny events", func() {
+		denyIntegrationTest(int64(14764643), test_data.CatAddress(), "0x13141b8a5e4a82ebc6b636849dd6a515185d6236", defaultOffset)
```

Comment:
super small, but I wonder if we could `"0x13141b8a5e4a82ebc6b636849dd6a515185d6236"` out into a var?